src/nvim/po/de.po

```diff
@@ -19,132 +19,375 @@ msgstr ""
+msgstr ""
+
+#: ../api/private/helpers.c:139
+#, c-format
+msgid "Key is fixed: %s"
+msgstr ""
+
+#: ../api/private/helpers.c:142
+#, fuzzy, c-format
+msgid "Key is locked: %s"
+msgstr "E741: Wert ist gesperrt: %s"
+
+#. Doesn't exist, fail
+#: ../api/private/helpers.c:151
+#, fuzzy, c-format
+msgid "Key \"%s\" doesn't exist"
+msgstr "Die Datei \"%s\" existiert nicht"
+
+#: ../api/private/helpers.c:205 ../api/private/helpers.c:260
+#, fuzzy
+msgid "Empty option name"
+msgstr "E792: Leerer Men�name"
+
+#: ../api/private/helpers.c:218 ../api/private/helpers.c:269
+#, fuzzy, c-format
+msgid "Invalid option name \"%s\""
+msgstr "E755: Ung�ltige Region in %s"
+
+#: ../api/private/helpers.c:237
+#, fuzzy, c-format
+msgid "Unable to get value for option \"%s\""
 msgstr "Kann Optionswert nicht ermitteln"
 
-#: ../api/private/helpers.c:200
-msgid "internal error: unknown option type"
-msgstr "interner Fehler: unbekannter Optionstyp"
+#: ../api/private/helpers.c:243
+#, fuzzy, c-format
+msgid "Unknown type for option \"%s\""
+msgstr "E355: Unbekannte Option: %s"
 
-#: ../cursor_shape.c:68
-msgid "E545: Missing colon"
-msgstr "E545: Fehlender Doppelpunkt"
+#: ../api/private/helpers.c:278
+#, fuzzy, c-format
+msgid "Unable to unset option \"%s\""
+msgstr "Kann Optionswert nicht ermitteln"
 
-#: ../cursor_shape.c:70 ../cursor_shape.c:94
-msgid "E546: Illegal mode"
-msgstr "E546: Unzul�ssiger Modus"
+#: ../api/private/helpers.c:284
+#, c-format
+msgid "Cannot unset option \"%s\" because it doesn't have a global value"
+msgstr "Option \"%s\" kann nicht abgestellt werden weil er nicht global ist"
 
-#: ../cursor_shape.c:134
-msgid "E548: digit expected"
-msgstr "E548: Ziffer erwartet"
+#: ../api/private/helpers.c:300
+#, c-format
+msgid "Option \"%s\" requires a boolean value"
+msgstr "Option \"%s\" ben�tigt ein Wert"
 
-#: ../cursor_shape.c:138
-msgid "E549: Illegal percentage"
-msgstr "E549: Unzul�ssige Prozentangabe"
+#: ../api/private/helpers.c:311
+#, c-format
+msgid "Option \"%s\" requires an integer value"
+msgstr "Option \"%s\" ben�tigt ein Ganzzahl"
 
-#: ../os/shell.c:184
-msgid ""
-"\n"
-"Cannot execute shell "
+#: ../api/private/helpers.c:319
+#, c-format
+msgid "Value for option \"%s\" is outside range"
+msgstr "Option \"%s\" ist au�erhalb des zul�ssigen Bereichs"
+
+#: ../api/private/helpers.c:330
+#, c-format
+msgid "Option \"%s\" requires a string value"
+msgstr "Option \"%s\" ben�tigt ein Zeichenfolge"
+
+#: ../api/private/helpers.c:563
+msgid "Invalid buffer id"
+msgstr "Ung�ltige Puffernummer"
+
+#: ../api/private/helpers.c:578
+msgid "Invalid window id"
+msgstr "Ung�ltige Fenster-ID"
+
+#: ../api/private/helpers.c:593
+msgid "Invalid tabpage id"
+msgstr "Ung�ltige Seitenreiter-ID"
+
+#: ../api/private/helpers.c:661
+msgid "Integer value outside range"
+msgstr "Ganzzahl au�erhalb des zul�ssigen Bereichs"
+
+#: ../api/private/helpers.c:716
+msgid "Empty dictionary keys aren't allowed"
+msgstr "Dictionary Schl�ssel k�nnen nicht leer sein"
```

Comment:
If we really find no suitable translation for dictionary, it should art least be hyphenated ('Dictionary-Schl.').
Wikipedia seems to suggest 'Assoziatives Array'...